Documentation/ceph-monitoring.md

```diff
@@ -81,7 +81,11 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
+To enable prometheus alerts during cluster deployment,
+```BASH
+kubectl create -f monitoring-roles.yaml
+``` 
+then, make following changes to `cluster.yaml`
```

Comment:
"To enable monitoring, make the following changes to `cluster.yaml`. Monitoring will be enabled during deployment."